app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -438,20 +438,20 @@ def get_image(img, b_name)
 
   # Determine if a button should be selected for buttonTwoState
   def twostate_button_selected(id)
-    return true if id.starts_with?("view_") && id.ends_with?("textual")  # Summary view buttons
-    return true if @gtl_type && id.starts_with?("view_") && id.ends_with?(@gtl_type)  # GTL view buttons
-    return true if @ght_type && id.starts_with?("view_") && id.ends_with?(@ght_type)  # GHT view buttons on report show
-    return true if id.starts_with?("tree_") && id.ends_with?(settings(:views, :treesize).to_i == 32 ? "large" : "small")
-    return true if id.starts_with?("compare_") && id.ends_with?(settings(:views, :compare))
-    return true if id.starts_with?("drift_") && id.ends_with?(settings(:views, :drift))
-    return true if id == "compare_all"
-    return true if id == "drift_all"
-    return true if id.starts_with?("comparemode_") && id.ends_with?(settings(:views, :compare_mode))
-    return true if id.starts_with?("driftmode_") && id.ends_with?(settings(:views, :drift_mode))
-    return true if id == "view_dashboard" && @showtype == "dashboard"
-    return true if id == "view_topology" && @showtype == "topology"
-    return true if id == "view_summary" && @showtype == "main"
-    false
+    (
```

Comment:
Heads up.. this bit won't get in ;)

I'm guessing there's an actual change hidden in there?

Reply:
ahh ... it's codeclimate 's fault :-) 
will revert ( but this will make codeclimate unhappy )

p.s.
I added a add-hoc-metrics twostate button :-)
https://github.com/ManageIQ/manageiq-ui-classic/pull/3296/files#diff-68677a7d5d33ee19b2fd378d150a1c73R444

Reply:
aah, thanks :)